src/openssl/lib_crypto.cr

```diff
@@ -174,8 +175,15 @@ lib LibCrypto
   fun evp_digestupdate = EVP_DigestUpdate(ctx : EVP_MD_CTX, data : UInt8*, count : LibC::SizeT) : Int32
   fun evp_md_ctx_copy = EVP_MD_CTX_copy(dst : EVP_MD_CTX, src : EVP_MD_CTX) : Int32
   fun evp_md_ctx_md = EVP_MD_CTX_md(ctx : EVP_MD_CTX) : EVP_MD
-  fun evp_md_size = EVP_MD_size(md : EVP_MD) : Int32
-  fun evp_md_block_size = EVP_MD_block_size(md : EVP_MD) : LibC::Int
+
+  {% if compare_versions(OPENSSL_VERSION, "3.0.0") >= 0 %}
```

Comment:
This could use a feature check instead.

ditto elsewhere

Reply:
How so?

Reply:
I meant checking for every new method signature via `LibCrypto.has_method?`, but perhaps, in this case, doesn't really make sense.

Reply:
This is the lib definiton. That's where these functions are defined.

Reply:
![image](https://i.giphy.com/media/kOIbusN7fPnkk/giphy.webp)